src/test/java/seedu/address/storage/JsonAdaptedAnimalTest.java

```diff
@@ -34,6 +41,8 @@ public class JsonAdaptedAnimalTest {
     private static final String VALID_DATE_OF_BIRTH = TOFU.getDateOfBirthForSerialization();
     private static final String VALID_ADMISSION_DATE = TOFU.getAdmissionDateForSerialization();
 
+    private static final List<JsonAdaptedTask> VALID_TASK_LIST = TOFU.getTaskListForSerialization();
```

Comment:
Is there an INVALID_TASK_LIST?

Reply:
I think there isn't an invalid task list, since a task list can be empty. For an Invalid task, this will be tested in the JsonAdaptedTaskTest